### ISSUE TITLE
currentViewModel is undefined in modelChanged in compose bindings.

### ISSUE THREAD

**Issue by Souldrinker** [aurelia/templating-resources#131]
In one of the Aurelia projects I've worked with that has been on hold for a while I had an issue back in February regarding the activate method not being called when the model attribute in a compose binding changes(https://github.com/aurelia/templating-resources/issues/15).

As far as I can remember I tested it and everything seemed fine after the bug fix on March 21st, but unfortunately that issue seem to have come back. I'm not sure when since I have been focusing on another project for a couple of months.

I'm not really sure where the issue is, but my compose binding looks like this:
```
<compose model.bind="currentBoardId" view-model.bind="boardView"></compose>
```
...where I set the currentBoardId (guid represented as string) and boardView (a string "views/portal/board/board-model") each time the user "changes tabs" in my application in which case I want the content below the tabs to change.

In my case the boardView gets set to the same value each time and thus doesn't change, but the currentBoardId changes everytime the user switches tab.

When I debug through the compose.js file I can see that the ```modelChanged```method gets called every time I switch tab, but ```_this.currentViewModel``` in the micro task callback is undefined and thus the activate method won't get called and nothing happens in my app.

When I change the code in the parent view model to do like this it works:
```
this.currentBoardId = boardId;
this.boardView = null;
setTimeout(() => this.boardView = "views/portal/board/board-model", 10);
```
...but that seems hacky and I'm rather sure the modelChange method has worked before by just changing the model and should work?

**Comment by EisenbergEffect** [comment]
I was inspecting the code myself yesterday and saw something that didn't
look quite right. I will take a closer look.

**Comment by EisenbergEffect** [comment]
I believe this was fixed in this commit: 90ce0d4cfd22212e5c37f689c1d0f93e655cee8f
